### DISCUSSION TITLE
Can we use gradle instead of maven?

### DISCUSSION THREAD

**Discussion by GalaxyShark** [ashishpawar517/AutoCompiler#25] [Q&A] [answered]
Mostly I have worked with gradle which seems friendly than maven? is there any specific reason you have chosen maven or can we move to gradle?

**Comment by ashishpawar517** [comment, marked as answer]
Yes I have also worked on Gradle, which is a nice build tool, I am revamping the autocompiler and VScode works well with Maven that's why I have chosen it, of course we can choose Gradle in future if we wise to

**Reply by GalaxyShark** [reply]
yeah that'll be great thanks for answering
